packages/terra-form-input/src/terra-dev-site/test/form-input/SupportedTypesInput.test.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import Input from '../../../Input';
+import styles from './common/Input.test.module.scss';
+
+const cx = classNames.bind(styles);
+
+const supportedTypes = [undefined, 'text', 'number', 'password', 'email', 'search', 'tel', 'url', 'hidden'];
+
+const getInputTypeExamples = (arr) => {
+  const inputTypeExamples = [];
+  for (let i = 0; i < arr.length; i += 1) {
+    inputTypeExamples.push((
+      <div className={cx('content-wrapper')}>
+        <label htmlFor={`supported-${arr[i]}-input`}>{`Input type=${arr[i]}`}</label>
+        <Input name={`supported-${arr[i]}-input`} type={arr[i]} ariaLabel={`Input type=${arr[i]}`} />
```

Comment:
The `ariaLabel` is redundant and not needed since these examples already include the preferred `label` element with the `htmlFor` implementation (something our other input tests & examples should have used instead, which we can fix on future enhancement of the full component). Other raw/standalone input examples for terra-form-input don't use a label element, instead they (1)either have no text above or (2)use a div with no connecting aria, so a quick workaround at the time was to use `ariaLabel`  on those freestanding (non-form-field) examples to pass axe checks, but using `aria-label` is not our Terra recommended A11Y implementation for inputs, instead using the explicit visible label as the preferred method.
- https://dequeuniversity.com/rules/axe/4.1/label-title-only
- https://www.deque.com/blog/anatomy-of-accessible-forms-best-practices/
- https://www.w3.org/WAI/tutorials/forms/labels/
- https://www.a11ywithlindsey.com/blog/introduction-accessible-labeling

Either can update to remove the ariaLabel:
``` diff
-        <Input name={`html-original-${arr[i]}-input`} type={arr[i]} ariaLabel={`Input type=${arr[i]}`} />
+        <Input name={`html-original-${arr[i]}-input`} type={arr[i]}  />
```

or less ideal, but would match existing examples, change the label to a div:
```
-        <label htmlFor={`html-original-${arr[i]}-input`}>{`Input type=${arr[i]}`}</label>
+        <div>{`Input type=${arr[i]}`}</div>
```

Reply:
I have updated wdio tests to be only for `supported` tests. And fixed the accessibility issue by replacing the `name` prop with `id`.